packages/common/src/ui/components/inputs/SearchBar/SearchBar.tsx

```diff
@@ -38,7 +40,7 @@ export const SearchBar: FC<SearchBarProps> = ({
       setLoading(false);
     },
     [onChange],
-    500
+    debounceTime
```

Comment:
Made the debounce time configurable (though leaving existing value as default). Seemed a bit weird seeing a loading spinner for a list filter that can be instant since we already have all the data -- have set debounceTime to `0` in that case.